test/test_action_response.ts

```diff
@@ -9,4 +10,24 @@ describe("ActionResponse validation", () => {
     chai.expect(jsonResponse.looker).to.not.be.null
     done()
   })
+
+  it("must populate errors object if provided", () => {
+    const errors: Errors = {
+      http_code: 500,
+      status_code: "TEST_FAIL",
+      message: "testing failure message",
+      location: "actions/test_action_response",
+      documentation_url: "http://test/documentation",
+    }
+    const response = new ActionResponse({success: false, errors: [errors]})
+    const jsonResponse = response.asJson()
+    chai.expect(jsonResponse.looker.errors[0]).to.equal(errors)
+  })
+
+  it("must be a valid response if errors object is not provided", () => {
+    const response = new ActionResponse({success: false})
+    const jsonResponse = response.asJson()
+    chai.expect(jsonResponse.looker).to.not.be.null
+    chai.expect(jsonResponse.looker.errors).to.be.undefined
```

Comment:
We should validate that success is false here as well, given that we're putting it in the object definition.